src/components/MonitorList.vue

```diff
@@ -204,8 +215,45 @@ export default {
             }
         },
     },
-    mounted() {
+    async mounted() {
```

Comment:
This seems a bit overcomplicated. 

If all relevant state is stored in the url, why is this necessary?

Reply:
If you mean the variables like `activeStates` and `statusStates`, they are used for an indirection, since names are used instead of IDs

Reply:
Yeah `async` was unnecessary right there